### PR TITLE
Fix permissions on workspace dir and add build workflow

### PR DESCRIPTION
**Pull request by elleFlorio**
Ok, I did a mistake and pushed to a wrong (and old) branch... forgive me 🙏 
Anyway, this PR fixes a possible problem with permissions inside the workspace folder.
Also adds a workflow to build the image on PRs/merge to master.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -12,7 +12,8 @@ RUN apk add --no-cache --virtual build-dependencies python-dev libffi-dev gcc mu
 RUN addgroup -S appgroup && adduser -S appuser -G appgroup
 
 USER appuser
-RUN mkdir -p /home/appuser/.dbt/
+RUN mkdir -p /home/appuser/.dbt/ && \
+    mkdir -p /home/appuser/workspace
```
Comment (alathon) [inline Dockerfile]: You don't need this instruction, `WORKDIR` creates the folder for you.

File: .github/scripts/get-latest-version.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```
Comment (alathon) [inline .github/scripts/get-latest-version.sh]: What's the need for auto-fetching the latest version, instead of specifying it explicitly?

File: Dockerfile (hunk 1 above)
Comment (elleFlorio) [reply Dockerfile]: Nope, I added that because `WORKDIR` creates the folder as `root`, I need `appuser` to be the owner of the folder.

File: .github/scripts/get-latest-version.sh (hunk 2 above)
Comment (elleFlorio) [reply .github/scripts/get-latest-version.sh]: I thought this was a convenient way to avoid manual version checking every time a PR is submitted. 😛

Comment (alathon) [reply .github/scripts/get-latest-version.sh]: I'd prefer we manually choose to update dbt or (more likely) don't, as its quite a young software :)

Comment (elleFlorio) [reply .github/scripts/get-latest-version.sh]: One thing to mention: this is not where we decide the dbt version to use. Here we just create the docker image of DBT, then it is on the service that uses it to specify the correct version. If you look at `hippocampus` you will see that it requires v0.15.2. Even if the latest version will be v0.16.0 and this workflow builds that image version, `hippocampus` will still use the other version until we explicitly say that we want the new base image. I would never set an auto-update on the version of the service we use... 😅
This repo is basically a self-maintaining repo that every Monday checks if there is a new dbt version and creates the new docker image if required.

TL;DR: nothing will break if we update dbt here. It is up to the service that uses the image to choose the version to use. Like any other Dockerised service: when you use it, you decide the version to use. 🙂

Comment (alathon) [reply .github/scripts/get-latest-version.sh:1]: Makes sense!
